RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 5.4
 -----
+- [*] PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
```

Comment:
This is great @shiki!
But could we put this on top of the doc, before the first version number (I mean, above `5.4` in this example)?
And could we use an easy to identify sequence at the beginning of the string (like `*** PLEASE FOLLOW ...`)?
This would make processing the file so much easier 😅

Reply:
@loremattei Thanks for the feedback! I updated it. How does it look? Will it be a problem for the deployment workflow/automation?